src/decisionengine/framework/modules/EmptySource.py

```diff
@@ -0,0 +1,32 @@
+"""
+This dummy source takes the name of a source datablock
+from config file as parameter "datablock_type" and produces
+an empty pandas DataFrame as a datablock with that name
+"""
+import pandas as pd
+
+from decisionengine.framework.modules import Source
+from decisionengine.framework.modules.Source import Parameter
+
+
+@Source.supports_config(
+    Parameter("datablock_type", default=""),
+)
+class EmptySource(Source.Source):
```

Comment:
Might call this `SingleProductSource` or something a little more descriptive.

Reply:
Hmmm, it creates/sources an empty data product. Isn't this descriptive enough?